### DISCUSSION TITLE
Intel HLS - ATLAS tracking project

### DISCUSSION THREAD

**Discussion by AleCamplani** [fastmachinelearning/hls4ml#658] [Ideas]
Hello! For the ATLAS Phase II upgrade, we would like to test the performance of a GNN (pythorch) for particle track reconstruction on an Intel accelerator card. This would imply the translation to Intel HLS.
I have seen your paper, where you have done it for Xilinx: https://arxiv.org/abs/2112.02048
And the code: https://github.com/fastmachinelearning/hls4ml/pull/379

I would like to ask you if we could arrange a chat (maybe next week?), as we would like to understand how to add the Intel feature :)
Thanks in advance.

Alessandra Camplani

**Comment by vloncar** [comment]
Hi Alessandra,

We're in the process of refactoring that implementation to fit more with the current codebase of hls4ml, and doing a port to Intel HLS in parallel would be great. We can fix a date for chat over email, I'll send one now.

Cheers, V.
